public/src/reducers/calls.jsx

```diff
@@ -1,37 +1,62 @@
 const defaultCalls = {
-  current_call: undefined,
-  next_call: undefined,
-  calls_made: 0
+  current_call: false,
```

Comment:
Is there a reason we don't just want to store current_call as an object in the store, instead of breaking it out like this and then reconstructing whenever we have to make a request?

Reply:
I'll explain more in person.